### PR TITLE
Update INL Application page

### LINKED ISSUE #22290: Update INL Applications web page to include links to hpcondemand sites
## Reason
We need a way to communicate where our internal documentation sites reside

## Design
Add applications to the existing INL Applications page: https://mooseframework.inl.gov/help/inl/applications.html

## Impact
A page of links to application documentation, and or other resources

### PR DESCRIPTION
**Pull request by milljm**
Update our application page with links to internal resources. Fix peacock tests when PyQt isn't available (freebie).

Closes #22290

### CODE REVIEW

**Comment by milljm** [comment]
And documentation will keep failing until my large_media PR gets merged: https://github.com/idaholab/large_media/pull/32

File: modules/doc/content/help/inl/applications.md [hunk 1]
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
...
+!col small=12 medium=4 large=4
+!media application_logos/Grizzly.png style=width:100%;
+!row-end!
+
+## MARMOT
+
+!row!
+!col small=12 medium=8 large=8
+[MARMOT Website](https://marmot-docs.hpcondemand.inl.gov/latest/)\\
+MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other code that works on engineering scale, which is larger than mesoscale, with an example being BISON. MARMOT solves equations involving solid mechanics and heat conduction using the finite element method.
```
Comment (joshuahansel) [inline modules/doc/content/help/inl/applications.md:49]: - to other applications that work at a larger scale, such as Bison, which works at an engineering scale.

File: modules/doc/content/help/inl/applications.md [hunk 2]
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
...
+
+!row!
+!col small=12 medium=8 large=8
+[Pronghorn Website](https://pronghorn-docs.hpcondemand.inl.gov/latest/)\\
+Pronghorn is a multi-dimensional, coarse-mesh, thermal-hydraulics code for advanced reactors and is particularly well-suited to model gas-cooled pebble bed and prismatic reactors. It serves the intermediate fidelity realm situated between detailed computational fluid dynamics analysis and lumped system models.
+
+!col small=12 medium=4 large=4
+!row-end!
+
+## Relap-7
```
Comment (joshuahansel) [inline modules/doc/content/help/inl/applications.md:65]: Capitalize RELAP

File: modules/doc/content/help/inl/applications.md [hunk 3]
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
...
+
+!row!
+!col small=12 medium=8 large=8
+[Relap-7 Website](https://relap7-docs.hpcondemand.inl.gov/latest/)\\
+A next generation nuclear systems safety code, RELAP-7 takes advantage of advances in computer architecture, software design, numerical methods, and physical models for use in the Risk Informed Safety Margin Characterization (RISMC) methodology and in nuclear power plant (NPP) safety analysis. RELAP-7 is a more capable than its predecessors in the RELAP family due to better flow models, improved numerical approximations, the ability to handle long duration events like full life cycle fuel evaluations, and easy coupling to other simulation code.
```
Comment (joshuahansel) [inline modules/doc/content/help/inl/applications.md:70]: Replace the whole description. I suggest:

RELAP-7 is a two-phase thermal systems code based on MOOSE's thermal hydraulics module. RELAP-7 provides two-phase components suitable for LWRs provides closures for two-phase water from TRACE.

File: modules/doc/content/help/inl/applications.md [hunk 4]
```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
...
+!row-end!
+
+<!-- Sabertooth goes here -->
+
+# Sockeye
+
+!row!
+!col small=12 medium=8 large=8
+[Sockeye Website](https://sockeye-docs.hpcondemand.inl.gov/latest/)\\
+Sockeye is a MOOSE-based heat pipe simulator and analysis tool. It, therefore, provides the ability to accurately predict heat transfer for heat-pipe-cooled microreactors and other heat pipe applications. Importantly, Sockeye models heat conduction transients in 1D and 2D as well as offering tools to analyze the operating envelope of heat pipes. So, it provides insight into operational limits in transient conditions, something not readily possible with steady-state analysis. Using Sockeye, users can spot operational limits and adjust designs accordingly.
```
Comment (joshuahansel) [inline modules/doc/content/help/inl/applications.md:83]: Replace the whole description. I suggest:

Sockeye is a heat pipe analysis application geared towards heat pipes of interest in heat-pipe-cooled microreactors. Sockeye provides various transient heat pipe modeling capabilities, including a 1D, two-phase flow model and a 2D effective thermal conductivity model based on heat conduction, which leverages analytic expressions of operational limits.

**Review by joshuahansel — CHANGES_REQUESTED** [review]

**Review by joshuahansel — APPROVED** [review]

**Pull request merged**